webapp/src/pages/Settings.tsx

```diff
@@ -679,13 +709,18 @@ const Settings: React.FC = () => {
         >
           {getAnalysesCustomizationSection()}
         </AccordionLayout>
-      </Paper>
-      <Box display="flex" justifyContent="space-between" paddingY={2}>
+      </DialogContent>
+      <DialogActions
+        sx={{
+          justifyContent: "space-between",
+        }}
+      >
```

Comment:
Detail, but that works on one line, to save a little bit of space.
```suggestion
      <DialogActions sx={{ justifyContent: "space-between" }}>
```